modules/core/src/main/scala/org/tessellation/modules/Storages.scala

```diff
@@ -5,31 +5,36 @@ import cats.syntax.flatMap._
 import cats.syntax.functor._
 
 import org.tessellation.domain.cluster.storage.AddressStorage
+import org.tessellation.domain.snapshot.GlobalSnapshotStorage
 import org.tessellation.domain.trust.storage.TrustStorage
 import org.tessellation.infrastructure.cluster.storage.AddressStorage
 import org.tessellation.infrastructure.db.Database
+import org.tessellation.infrastructure.snapshot.GlobalSnapshotStorage
 import org.tessellation.infrastructure.trust.storage.TrustStorage
+import org.tessellation.kryo.KryoSerializer
 import org.tessellation.sdk.domain.cluster.storage.{ClusterStorage, SessionStorage}
 import org.tessellation.sdk.domain.gossip.RumorStorage
 import org.tessellation.sdk.domain.node.NodeStorage
 import org.tessellation.sdk.modules.SdkStorages
 
 object Storages {
 
-  def make[F[_]: Async: Database](
+  def make[F[_]: Async: Database: KryoSerializer](
     sdkStorages: SdkStorages[F]
   ): F[Storages[F]] =
     for {
       addressStorage <- AddressStorage.make[F]
       trustStorage <- TrustStorage.make[F]
+      globalSnapshotStorage <- GlobalSnapshotStorage.make[F](org.tessellation.infrastructure.snapshot.genesis)
```

Comment:
can you import `org.tessellation.infrastructure.snapshot.genesis` and pass `genesis` here only?